app/app.go

```diff
@@ -321,6 +322,11 @@ func wireCoreWorkflow(ctx context.Context, life *lifecycle.Manager, conf Config,
 		pubshares = append(pubshares, eth2Share)
 	}
 
+	peers, err := lock.Peers()
```

Comment:
i think `peers` is needed as a param for `wireTracker` so why not colocate them?

Reply:
in general yes, but this is such low signal code, I put it at the start with all the "prep" variables.